src/SIL.Harmony.Tests/ResourceTests/RemoteResourcesTests.cs

```diff
@@ -0,0 +1,139 @@
+﻿using System.Runtime.CompilerServices;
+using Microsoft.Extensions.DependencyInjection;
+using SIL.Harmony.Resource;
+
+namespace SIL.Harmony.Tests.ResourceTests;
+
+public class RemoteResourcesTests : DataModelTestBase
```

Comment:
In your tests you always know before hand if you're dealing with a local or remote resource.

I feel like maybe there should be a `public Resource GetResource()` method where `Resource` aggregates the local and remote resource looks something like:

```casharp
Resource
{
Id,
LocalPath: string?,
RemoteId: string?,
IsLocal: bool,
IsRemote: bool,
}
```

I suppose a Harmony user could implement it themselves. A join is probably trivial if projected tables are enabled.

Reply:
yeah that's a good idea

Reply:
I think there was a slight misunderstanding.

You added a `GetResources()` (plural). My suggestion was for looking up a **single** resource.
I'm thinking about the case where I have a specific Resource ID, but I don't know if it's local or remote (or both), so I'd like to be able to get a `CrdtResource` just for that one resource.

Can we do the join in SQL? I guess it doesn't matter for `GetResources()` (plural), because everything ends up in memory anyway.

Also I wonder if `CrdtResource` is a misleading name: the local-resources are explicitly non-crdt values, because they're local only. 🤔

Reply:
Ahh ok, that makes sense, yeah I can easily make that API (probably on top of the one I just made for now).

As for the name. Yeah that's true, I don't want to use just Resource as the class name as that's way too generic and there's other dotnet classes with the same name so I don't want the confusion. So maybe HarmonyResource?

Reply:
Yeah, you're right `Resource` is heavily used 😆.
I like `HarmonyResource` except for the fact that it would be the first type to use the word `Harmony` in it. 🤔

Some more ideas:
- `UnifiedResource`
- `HybridResource`

Hybrid sounds coolest, but I think it almost sounds too cool and fancy.
Personally, I think I'd opt for `UnifiedResource`.

Reply:
I've changed the name to Harmony resource and created #22 to change our names from Crdt- to Harmony